src/Samples/Toolkit.SampleApp.UWP/Samples/Forms/FeatureFormViewSample.xaml

```diff
@@ -0,0 +1,48 @@
+﻿<Page
+    x:Class="Esri.ArcGISRuntime.Toolkit.SampleApp.Samples.Forms.FeatureFormViewSample"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+    xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+        <esri:MapView x:Name="mapView"
+               Map="{x:Bind Map}"
+               GeoViewTapped="mapView_GeoViewTapped"/>
+
+        <Border HorizontalAlignment="Center" VerticalAlignment="Top" Background="#99ffffff" Padding="5" Margin="10" BorderThickness="1" BorderBrush="Black">
+            <TextBlock Text="Click a feature to open its FeatureFormView" FontSize="18" />
```

Comment:
Sample: This text is unreadable (appears white-on-white) when using dark mode.

![2024-08-02_001613 Toolkit SampleApp WinUI](https://github.com/user-attachments/assets/2f89c6c8-44a0-4e75-9729-3ac7874bc2ce)